x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java

```diff
@@ -753,7 +753,8 @@ public void testKibanaSystemRole() {
             ".fleet-enrollment-api-keys",
             ".fleet-policies",
             ".fleet-actions-results",
-            ".fleet-servers"
+            ".fleet-servers",
+            ".fleet-fileds"
```

Comment:
Just for my own knowledge - why did we name this `fileds`?

Reply:
these are now datastreams, it was done to differentiate from the legacy `.fleet-file` which were regular indices.